hyperbahn-client.js

```diff
@@ -175,6 +176,26 @@ HyperbahnClient.prototype.setReportTracing = function setReportTracing(bool) {
     self.reportTracing = bool;
 };
 
+HyperbahnClient.prototype.getThriftCodecSync =
```

Comment:
Agree on codec being unnecessary jargon. `getThriftSync()`, `getThriftFile()`, `readThriftFile()`, etc. are all better. Also, get for me implies some sort of network fetching. For local disk i/o, `read` is preferable to `get` unless we plan to allow `options.thriftFile` to be something other than a local filepath.